tiscale_consts.py

```diff
@@ -29,7 +33,8 @@
 TISCALE_TEST_PDF_FILE = "tiscale_test_connectivity.pdf"
 TISCALE_SLEEP_SECS = 10
 TISCALE_MSG_REPORT_PENDING = "Report Pending"
-TISCALE_MSG_MAX_POLLS_REACHED = "Reached max polling attempts. Please use the MD5 or Sha256 of the file as a parameter to <b>get report</b> to query the report status."
+TISCALE_MSG_MAX_POLLS_REACHED = """Reached max polling attempts.
+Please use the MD5 or Sha256 of the file as a parameter to <b>get report</b> to query the report status."""
```

Comment:
```suggestion
TISCALE_MSG_MAX_POLLS_REACHED = "Reached max polling attempts. " \
    "Please use the MD5 or Sha256 of the file as a parameter to <b>get report</b> to query the report status."
```
a multiline string would preserve the literal spacing in the printed string